src/test/java/pages/stepDefinitions/LoginStepDefinitions.java

```diff
@@ -0,0 +1,82 @@
+package pages.stepDefinitions;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import pages.LoginPage;
+
+import java.time.Duration;
+
+public class LoginStepDefinitions {
+
+
+    WebDriver driver;
+    WebDriverWait wait;
+
+    @Before
+    public void openBrowser() {
+        WebDriverManager.firefoxdriver().setup();
+        ChromeOptions options = new ChromeOptions();
+        driver = new ChromeDriver();
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+
+    }
+
+    @After
+    public void iCloseTheBrowser() {
+        driver.quit();
+
+    }
+
+    @Given("I open Login Page")
+    public void openBrowser() {
+        driver = new ChromeDriver();
+        wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+
+        driver.get("https://bbb.testpro.io");
+    }
+
+    @When("I enter email {string}")
+    public void iEnterEmail(String email) {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.provideEmail(email);
+        //wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[type='email']")));
+
+    }
+
+    @And("I enter password {string}")
+    public void iEnterPassword(String password) {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.provideEmail(password);
+        //wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[type='password']]")));
+
+    }
+
+    @And("I submit")
+    public void iSubmit() {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.clickSubmitButton();
+        //wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[type='submit']")));
+
+    }
+
+    @Then("I am logged in")
+    public void iAmLoggedIn() {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.isEmailFieldVisible();
+        //wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("img.avatar"))).isDisplayed());
+
+    }
+
+
+}
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/227901732-a569b740-d5a5-4477-9dfd-75957b9800d2.png)